src/async_perturb_test.py

```diff
@@ -323,11 +350,17 @@ def write_single_graph_details(state_calc_obj: CellCycleStateCalculation, it_cnt
     )
     print(f"Avg score for {organism} is {avg_score} for {it_cnt} iterations.")
 
+#Main method
 
 if __name__ == "__main__":
     start_time = time()
-    organism = "model01"
+    if len(sys.argv) != 6:
+        print("Incorrect format. Use: python async_perturb_test.py <model_name> <filter_states> <custom_state> <single_it_count> <double_it_count>")
+        exit(0)
+    #Change this variable to change the model
+    organism = sys.argv[1]
```

Comment:
Try to implement keyword-based argparser here and other places where `sys.argv` is used. Also give some default value if user does not pass any value in the arguments.